src/prefs/DevicePrefs.cpp

```diff
@@ -203,6 +206,40 @@ void DevicePrefs::PopulateOrExchange(ShuttleGui & S)
       S.EndThreeColumn();
    }
    S.EndStatic();
+
+   if (mProject != nullptr)
+   {
+      S.StartStatic(XO("Quality"));
+      {
+         S.StartThreeColumn();
+
+         auto cmbProjectRate = S.AddCombo(XO("Project rate:"), {}, {});
+
+         for (auto i = 0; i < AudioIOBase::NumStandardRates; ++i)
+            cmbProjectRate->Append(
+               XO("%d").Format(AudioIOBase::StandardRates[i]).Translation());
```

Comment:
Why use XO(“%d”)?  For locale dependent decimal point, or maybe even other digit characters.

Reply:
Maybe more places need to do that

Reply:
Unfortunate leftover, I had `%d Hz` before.

Reply:
Which makes more obvious sense as needing i18n

Reply:
I wonder sometimes though, should XO("%d") be a thing? https://en.wikipedia.org/wiki/Devanagari_numerals

We still use sexagesimal time units after all https://en.wikipedia.org/wiki/Cuneiform_Numbers_and_Punctuation

Reply:
Ideally any formatting function should handle the locale. But it seems to be not the case for wxString::Format